File: wait_formula/wait_forms/wait.py
from sympy import *

r, eta, beta, rho, a, h = symbols('r eta beta rho a h', real=True)
delta_Z = (-eta)/((2*pi*sin(beta*h)**2)) * (Integral(exp(-1j*2*beta*r)/rho, (rho, a, oo)) - 1*2*cos(beta*h)*Integral(exp(-beta*1j*(r + rho))/rho, (rho, a, oo)) - cos(beta*h)**2*Integral(exp(-1j*2*beta*rho)/rho, (rho, a, oo)))

File: wait_formula/wait_forms/wait.m
syms r eta beta rho a h real

r = sqrt(rho^2 + h^2)
delta_Z = (-eta)/((2*pi*sin(beta*h)^2)) * (int(exp(-1i*2*beta*r)/rho, rho, a, inf) - 1*2*cos(beta*h)*int(exp(-beta*1i*(r + rho))/rho, rho, a, inf) - cos(beta*h)^2*int(exp(-1i*2*beta*rho)/rho, rho, a, inf))
